etc/ci-script.sh

```diff
@@ -15,11 +15,25 @@ bunzip2 stackage-curator.bz2
 chmod +x stackage-curator
 mv stackage-curator ~/.local/bin
 
+# Get new stackage-curator
+CURATOR2=stackage-curator-2-9132b4b184f1f7f27c4c80ec0609460cea22ae75
+wget  "https://s3.amazonaws.com/www.snoyman.com/stackage-curator-2/$CURATOR2.bz2"
+bunzip2 "$CURATOR2.bz2"
+chmod +x $CURATOR2
+mv $CURATOR2 ~/.local/bin/stackage-curator-2
+
 # Install GHC
 stack setup $GHCVER
 
 # Update the index
 stack update
 
 # Check
-exec stack --resolver ghc-$GHCVER exec stackage-curator check
+stack --resolver ghc-$GHCVER exec stackage-curator check
```

Comment:
I think it makes more sense to run the new curator check as a separate component of the build matrix, that way if something is incompatible with only one of the tools we get a clearer indication of what failed.

Reply:
Yes, I had some gut feeling about the approach I used, you nailed it perfectly

Reply:
Done, could be seen at https://travis-ci.org/commercialhaskell/stackage/builds/485256063 - old check succeeds while the new one fails because of bounds constraints